edgeai-apps-utils/suite/bookworm/debian/control

```diff
@@ -0,0 +1,26 @@
+Source: edgeai-apps-utils
+Section: libs
+Priority: optional
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: cmake, debhelper-compat (= 13)
+Standards-Version: 4.5.1
+Homepage: https://git.ti.com/cgit/edgeai/edgeai-apps-utils
+Rules-Requires-Root: no
+
+Package: libedgeai-apps-utils0.1.0
```

Comment:
Shouldn't have versions in the package name unless we expect to have multiple version installed at the same time.